cmd/repl/prompt.go

```diff
@@ -236,6 +292,7 @@ func runPrompt(cb func(string)) {
 		prompt.OptionPrefix(state.prefix),
 		prompt.OptionPrefixTextColor(prompt.DefaultColor),
 		prompt.OptionTitle("SQLFlow"),
+		prompt.OptionCompletionWordSeparator(" ="),
```

Comment:
Do we need a blank before `=`?

Reply:
This is to make sure completion stop at whitespace or a `=`